### ISSUE TITLE
LevelZ Compatibility

### ISSUE THREAD

**Issue by Zophix** [jexline/exline-mods-issues#6]
Hi! Would it be possible to add the compatibility with LevelZ by making some changes on the material class for copper?

Here are the suggestions made by the developer of LevelZ:

The dev has to change the material to an enum
The dev can override the toString method inside his material class like here: https://github.com/GitPois1x/DragonLoot/blob/master/src/main/java/net/dragonloot/item/DragonToolMaterial.java
The issue is here to know more if needed:
Globox1997/LevelZ#53

**Comment by jexline** [comment]
Yes, I will try to add this soon. I will test it out and let you know when I get it working.
Are you using 1.18 or 1.17? 
I will address the issue for all versions, but will start with 1.18 unless you are using different version.

**Comment by jexline** [comment]
I updated the mod to have the toString method overridden and using "COPPER" for the name.
However, I did not make it an enum tho, as the DragonTool mod above did not have it as one. 
I have LevelZ installed and will see if I can get it working soon!
EDIT: I've been trying to get it to work but am having trouble. I do not have any experience with datapacks and setting them up, so I need help to lock the copper tools in LevelZ.
